recipes/mongo-cxx-driver/all/conanfile.py

```diff
@@ -119,6 +119,9 @@ def package_info(self):
         # Need to ensure mongocxx is linked before bsoncxx
         self.cpp_info.libs = sorted(tools.collect_libs(self), reverse=True)
         self.cpp_info.includedirs.extend([os.path.join("include", lib, "v_noabi") for lib in ["bsoncxx", "mongocxx"]])
+        self.cpp_info.names["pkg_config"] = "libmongocxx"
+        self.cpp_info.names["cmake_find_package"] = "libmongocxx"
+        self.cpp_info.names["cmake_find_package_multi"] = "libmongocxx"
```

Comment:
Nop:
- mongocxx: https://github.com/mongodb/mongo-cxx-driver/blob/5b4628114ebddc9bd0092ea0eb3d66beef7818e9/src/mongocxx/CMakeLists.txt#L232 + https://github.com/mongodb/mongo-cxx-driver/blob/5b4628114ebddc9bd0092ea0eb3d66beef7818e9/cmake/MongocxxUtil.cmake#L46
- bsoncxx: https://github.com/mongodb/mongo-cxx-driver/blob/5b4628114ebddc9bd0092ea0eb3d66beef7818e9/src/bsoncxx/CMakeLists.txt#L205 + https://github.com/mongodb/mongo-cxx-driver/blob/5b4628114ebddc9bd0092ea0eb3d66beef7818e9/cmake/BsoncxxUtil.cmake#L58

Reply:
I think I did it.